src/extension.ts

```diff
@@ -31,6 +32,13 @@ export function activate(context: ExtensionContext) {
         },
     );
 
+    const refreshTreeView = commands.registerCommand(
+        "dodona.treeview.refresh",
+        async () => {
```

Comment:
Does not need to be async I think

```suggestion
        () => {
```